Gitlab importation - unhandled note type

When trying to import issues from Gitlab, it starts and then:

```
panic: unhandled note type

goroutine 1 [running]:
github.com/MichaelMure/git-bug/bridge/gitlab.(*gitlabImporter).ensureNote(0xc420290080, 0xc4201edfc0, 0xc4201a8aa0, 0xc42058a180, 0x0, 0x0)
```

It still successfully imported some bugs. :)

The failing bug may be titled with some accents, quotes and parentheses, I don't know if this could be the cause of the crash.

--- a/bridge/gitlab/import.go
+++ b/bridge/gitlab/import.go
@@ -248,7 +248,15 @@ func (gi *gitlabImporter) ensureNote(repo *cache.RepoCache, b *cache.BugCache, n
 
 		return err
 
-	case NOTE_UNKNOWN:
+	case NOTE_UNKNOWN,
+		NOTE_ASSIGNED,
+		NOTE_UNASSIGNED,
+		NOTE_CHANGED_MILESTONE,
+		NOTE_REMOVED_MILESTONE,
+		NOTE_CHANGED_DUEDATE,
+		NOTE_REMOVED_DUEDATE,
+		NOTE_LOCKED,
+		NOTE_UNLOCKED:
 		return nil
 
 	default:

